reco_utils/recommender/deeprec/config/asvd.yaml

```diff
@@ -8,7 +8,7 @@ data:
 #model
 model:
     method : classification # classification or regression
-    model_type : ASVD
+    model_type : A2SVD
```

Comment:
Can we rename all 'ASVD' to 'A2SVD' in this file, such as in MODEL_DIR and SUMMARIES_DIR?

Reply:
changed